src/basic/missing_capability.h

```diff
@@ -34,6 +42,9 @@
 #    undef CAP_LAST_CAP
 #  endif
 #endif
+
 #ifndef CAP_LAST_CAP
-#  define CAP_LAST_CAP SYSTEMD_CAP_LAST_CAP
+#    define CAP_LAST_CAP SYSTEMD_CAP_LAST_CAP
+#else
+assert_cc(CAP_LAST_CAP == SYSTEMD_CAP_LAST_CAP);
```

Comment:
This one has a special check above, which conflicts with this when CAP_LAST_CAP > SYSTEMD_CAP_LAST_CAP? We'll emit a warning but then trigger assertion?

Reply:
I removed the addition.